nav2_costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -112,13 +115,11 @@ InflationLayer::updateBounds(
     last_min_y_ = *min_y;
     last_max_x_ = *max_x;
     last_max_y_ = *max_y;
-    // For some reason when I make these -<double>::max() it does not
-    // work with Costmap2D::worldToMapEnforceBounds(), so I'm using
-    // -<float>::max() instead.
-    *min_x = -std::numeric_limits<float>::max();
-    *min_y = -std::numeric_limits<float>::max();
-    *max_x = std::numeric_limits<float>::max();
-    *max_y = std::numeric_limits<float>::max();
+
+    *min_x = std::numeric_limits<double>::lowest();
```

Comment:
lowest appears to just be -max ( https://en.cppreference.com/w/cpp/types/numeric_limits). Not sure what issue you were referring to that asked for that to be changed. Can you clarify / link?

Reply:
https://github.com/ros-planning/navigation2/blob/ecc42fb3879d470590b9d9ef893c7565cee3b69b/nav2_costmap_2d/plugins/inflation_layer.cpp#L115-L117